src/python/pants/backend/go/pkg.py

```diff
@@ -248,15 +282,68 @@ async def resolve_go_package(
     )
 
 
+@rule
+async def resolve_external_go_package(
+    request: ResolveExternalGoPackageRequest,
+) -> ResolvedGoPackage:
+    wrapped_target = await Get(WrappedTarget, Address, request.address)
+    target = wrapped_target.target
+
+    import_path = target[GoImportPath].value
+    if not import_path:
+        raise ValueError(
+            f"_go_ext_mod_package at address {request.address} does not have an import path set."
+        )
+
+    module_path = target[GoExternalModulePath].value
+    if not module_path:
+        raise ValueError(f"_go_ext_mod_package at address {request.address} has a blank `path`")
```

Comment:
You should override `compute_value()` to validate this there

Reply:
TODO